jsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["lib/**/*.js", "extensions/**/*.js"],
+  "include": ["packages/**/*.js"],
   "compilerOptions": {
     "checkJs": true,
     "module": "commonjs",
```

Comment:
Just flagging as I see it that you have es2019 in here, and 2018 in eslint.

The rest of the Ghost ecosystem is currently targetting 2018, although I have no strong feels about this.

Also does `moduleResolution: node` mean you don't need this:

``` 
 "exclude": [
    "node_modules"
  ]
```

(I'm comparing against the Utils repo)

Reply:
forgot to update the eslint version 😬 

Reason I put 2019 here is that 2019 is the newest version supported by node 12 (the min version of node 12 we're on has 100% of es2019 features implemented) - figured since we're doing a decent rewrite anyways it would make sense to use as new features as we could

Reply:
> Also does moduleResolution: node mean you don't need this:

to be honest I'm not sure `exclude: node_modules` is even needed 🤔 Given that jsconfig.json is essentially the same as tsconfig.json, but without the enforcement/compilation steps, I'm not sure the exclude functionality really does anything, unless you ended up opening a file from node_modules in your editor.